invenio/legacy/websession/webgroup.py

```diff
@@ -748,13 +748,13 @@ def perform_request_reject_member(uid,
                 group_name=group_name,
                 accepted=0,
                 ln=ln)
-            (body, dummy, dummy) = perform_request_send(
-                uid,
-                msg_to_user=user,
-                msg_to_group="",
-                msg_subject=msg_subjet,
-                msg_body=msg_body,
-                ln=ln)
+            # (body, dummy, dummy) = perform_request_send(
+            #     uid,
+            #     msg_to_user=user,
+            #     msg_to_group="",
+            #     msg_subject=msg_subjet,
+            #     msg_body=msg_body,
+            #     ln=ln)
```

Comment:
Isn't it simply better to kill it for the good? As anyway it will stay in git history?